packages/tocco-ui/src/SignalBox/StyledSignalBox.js

```diff
@@ -0,0 +1,44 @@
+import {theme} from 'styled-system'
+import styled from 'styled-components'
+
+import {StyledSignalListItem} from '../SignalList'
+import {stylingCondition} from '../utilStyles'
+
+// TODO icon should have line-height 1.4
+const StyledSignalBox = styled.div`
+  && {
+    background-color: ${props => getSignalBg(props)};
+    padding: ${theme('space.5')};
+    margin-bottom: ${theme('space.5')};
+    border-radius: ${theme('radii.2')};
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    ${StyledSignalListItem} {
+      color: inherit;
+    }
+  }
+`
+
+const getSignalBg = props => {
+  switch (props.condition) {
+    case stylingCondition.DANGER:
+      return theme('colors.signal.dangerBg')(props)
+    case stylingCondition.SUCCESS:
+      return theme('colors.signal.successBg')(props)
+    case stylingCondition.WARNING:
+      return theme('colors.signal.warningBg')(props)
+    case stylingCondition.PRIMARY:
+      return theme('colors.primary.fill.0')(props)
+    case stylingCondition.BASE:
+      return theme('colors.base.fill.1')(props)
+    default:
+      // eslint-disable-next-line no-console
+      console.warn('Be explicit by adding a case. You may want colorize the content.')
+      return 'inherit'
+  }
```

Comment:
please consider my comments in the pull request https://github.com/tocco/tocco-client/pull/451 here as well